stable/datadog/templates/container-trace-agent.yaml

```
@@ -21,9 +21,9 @@
     {{- if .Values.datadog.nonLocalTraffic }}
     - name: DD_DOGSTATSD_NON_LOCAL_TRAFFIC
       value: {{ .Values.datadog.nonLocalTraffic | quote }}
-    - name: DD_APM_NON_LOCAL_TRAFFIC
-      value: {{ .Values.datadog.nonLocalTraffic | quote }}
     {{- end }}     
+    - name: DD_APM_NON_LOCAL_TRAFFIC
+      value: {{ .Values.datadog.apm.enabled | quote }}
```

same here you can put True directly
```suggestion
      value: {{ true | quote }}
```